frontend/lib/Views/AdminPage/main.dart

```diff
@@ -0,0 +1,123 @@
+import 'package:adpv_frontend/Views/AdminPage/groups_view.dart';
+import 'package:flutter/material.dart';
+
+class AdminMainView extends StatefulWidget {
+  const AdminMainView({Key? key}) : super(key: key);
+
+  @override
+  State<AdminMainView> createState() => _AdminMainViewState();
+}
+
+class _AdminMainViewState extends State<AdminMainView> {
+  @override
```

Comment:
Czy to nie może być `StatelessWidget`? Bo w tym widget'cie nie zmieniamy nigdzie stanu, tylko mamy dwa przyciski, które zawsze będą takie same
cc: @kkoz3434

Reply:
exactly